jquery.swiftype.autocomplete.js

```diff
@@ -119,6 +118,16 @@
         }
       };
 
+      $this.showNoResults = function () {
```

Comment:
Just to be clear, there's no corresponding `hideNoResults` because we're `$list.empty();` elsewhere?

Reply:
yep. I couldn't think of a better name than `showNoResults`